src/script/conversation/MessageRepository.ts

```diff
@@ -809,9 +808,7 @@ export class MessageRepository {
       }
       return result;
     } catch (error) {
-      if (isBackendError(error)) {
-        await this.updateMessageAsFailed(conversation, payload.messageId, error);
-      }
+      await this.updateMessageAsFailed(conversation, payload.messageId, error);
```

Comment:
I could alternatively check for Axios errors with `isAxiosError(error)` but I think we would want to treat any error as a failure to send with the option to retry